Source/PeleLMeX_Utils.cpp

```diff
@@ -968,7 +968,10 @@ PeleLM::deriveComp(const std::string& a_name, Real a_time, int lev, int nGrow)
     // Get pressure: TODO no fillpatch for pressure just yet, simply get new
     // state
     auto* ldata_p = getLevelDataPtr(lev, AmrNewTime);
-    std::unique_ptr<MultiFab> reactmf = fillPatchReact(lev, a_time, nGrow);
+    std::unique_ptr<MultiFab> reactmf;
+    if (m_incompressible == 0) {
```

Comment:
This check should be on `m_do_react != 0`, otherwise multispecies nonreacting simulations will have the same segfault.